modules/argo-cd-command-line.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="command-line-tool_{context}"]
+= Argo CD CLI tool
+
+[role="_abstract"]
+The Argo CD CLI tool is a tool used to configure Argo CD through the command line. {gitops-title} does not support this binary. We recommend the users to use the OpenShift Console to configure the Argo CD.
```

Comment:
```suggestion
The Argo CD CLI tool is a tool used to configure Argo CD through the command line. {gitops-title} does not support this binary. Use the OpenShift Console to configure the Argo CD.
```
This is a tentative suggestion caused by https://redhat-documentation.github.io/supplementary-style-guide/
![image](https://user-images.githubusercontent.com/78564304/159700571-dcecef37-6216-4778-9720-c649cf02ccf1.png)